dist/ng-showdown.js

```diff
@@ -100,36 +119,62 @@
       }
 
       /**
-       * AngularJS Directive to Md to HTML transformation
+       * @deprecated
+       * Legacy AngularJS Directive to Md to HTML transformation
        *
        * Usage example:
        * <div sd-model-to-html="markdownText" ></div>
        *
        * @param {showdown.Converter} $showdown
+       * @param {$sanitize} $sanitize
        * @param {$sce} $sce
        * @returns {*}
        */
-      function markdownToHtmlDirective($showdown, $sce) {
+      function sdModelToHtmlDirective($showdown, $sanitize, $sce) {
         return {
           restrict: 'A',
-          link: link,
+          link: getLinkFn($showdown, $sanitize, $sce),
           scope: {
             model: '=sdModelToHtml'
           }
         };
+      }
+
+      /**
+       * AngularJS Directive to Md to HTML transformation
+       *
+       * Usage example:
+       * <div markdown-to-html="markdownText" ></div>
+       *
+       * @param {showdown.Converter} $showdown
+       * @param {$sanitize} $sanitize
+       * @param {$sce} $sce
+       * @returns {*}
+       */
+      function markdownToHtmlDirective($showdown, $sanitize, $sce) {
+        return {
+          restrict: 'A',
+          link: getLinkFn($showdown, $sanitize, $sce),
+          scope: {
+            model: '=markdownToHtml'
+          }
+        };
+      }
 
-        function link(scope, element) {
+      function getLinkFn($showdown, $sanitize, $sce) {
+        return function (scope, element) {
           scope.$watch('model', function (newValue) {
-            var val;
+            var val,
+                showdownHTML;
             if (typeof newValue === 'string') {
-              var showdownHTML = $showdown.makeHtml(newValue);
-              val = $sce.trustAsHtml(showdownHTML);
+              showdownHTML = $showdown.makeHtml(newValue);
+              val = ($showdown.getOption('sanitize')) ? $sanitize(showdownHTML) : $sce.trustAsHtml(showdownHTML);
```

Comment:
On a second thought: The difference might be that I use the full jQuery instead of the Angular jQuery light implementation. Has Angular's .element() different behavior here?